core/src/test/scala/org/apache/spark/scheduler/DAGSchedulerSuite.scala

```diff
@@ -517,6 +518,32 @@ class DAGSchedulerSuite extends SparkFunSuite with LocalSparkContext with Timeou
     assertDataStructuresEmpty()
   }
 
+  test("unserializable partition") {
```

Comment:
Can you change this to describe more precisely what's being tested? Something like "Scheduler aborts stages that have unserializable tasks"?